e2e/interchaintestv8/relayer_test.go

```diff
@@ -994,4 +1009,26 @@ func (s *RelayerTestSuite) ICS20TimeoutFromCosmosTimeoutTest(
 			s.Require().Equal(testvalues.InitialBalance, resp.Balance.Amount.Int64())
 		}))
 	}))
+
+	s.Require().True(s.Run("Verify no balance on Ethereum", func() {
+		denomOnEthereum := transfertypes.NewDenom(transferCoin.Denom, transfertypes.NewHop(transfertypes.PortID, testvalues.CustomClientID))
+
+		_, err := s.ics20Contract.IbcERC20Contract(nil, denomOnEthereum.Path())
+		// Ethereum side did not received the packet, ERC20 contract corresponding to the denom does not exist
+		s.Require().Error(err)
+		s.Require().Contains(err.Error(), "execution reverted: custom error 0xe1275e2f")
```

Comment:
I think we should remove this since each time we update the contracts, the error identifiers might change. I don't wanna have to maintain this line
```suggestion
```